tremor-script/rebar.config

```diff
@@ -2,32 +2,35 @@
 
 {plugins, [rebar3_auto]}.
 {cover_enabled, true}.
-{deps, [ lager, jsxd, jsx ]}.
+{deps, [lager, jsxd, jsx]}.
 
-{pre_hooks,
-  [{"(linux|darwin|solaris)", compile, "make -C c_src"},
-   {"(freebsd)", compile, "gmake -C c_src"}]}.
-{post_hooks,
-  [{"(linux|darwin|solaris)", clean, "make -C c_src clean"},
-   {"(freebsd)", clean, "gmake -C c_src clean"}]}.
+{pre_hooks, [
+  {"(linux|darwin|solaris|freebsd)", compile,
+        "sh -c \"cd .. && cargo build -p tremor-script-nif && cp target/debug/libtremor.so tremor-script/priv\""}
+]}.
 
-{port_specs, [{"priv/tremor_script.so",["c_src/*.c"]}]}.
+{port_specs, [{"priv/libtremor.so", ["c_src/*.c"]}]}.
```

Comment:
this should also have a platform specific suffix, no?

Reply:
we only compile for unix systems so not needed

Reply:
but we also compile for darwin, there it needs to be .dylib, no?

Reply:
so far this wasn't an issue but also the line is from the old code not the new one and can go away - which solves the problem one way or the other :D

Reply:
ah yeah, remove it then. :D